#Request
float64 width
float64 effort
---
#Response
bool success